tsdat/io/readers/zip_reader.py

```diff
@@ -92,6 +92,10 @@ def read(self, input_key: str) -> Dict[str, xr.Dataset]:
         else:
             fileobj = input_key
 
+        # TODO: Python has a built-in function, [`zip`]
```

Comment:
Nice catch. Maybe `zip_file` or `archive`?

Reply:
Resolved in 136e373bc4c9681e38e396d4ddb016c42c434d47.